xml/System.Buffers/SequenceReader`1.xml

```diff
@@ -801,11 +809,11 @@
         <Parameter Name="advancePastDelimiter" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="sequence">To be added.</param>
-        <param name="delimiters">To be added.</param>
-        <param name="advancePastDelimiter">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="sequence">The read data, if any.</param>
+        <param name="delimiters">The delimiters to look for.</param>
+        <param name="advancePastDelimiter"><see langword="true" /> to move past the first found instance of any of the given <paramref name="delimiters" />.</param>
+        <summary>Tries to read everything up to the given <paramref name="delimiters" />.</summary>
+        <returns><see langword="true" /> if any of the <paramref name="delimiters" /> were found.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if any of the <paramref name="delimiters" /> were found; otherwise, <see langword="false" />.</returns>
```